### PR TITLE
Update the issue template for new way of getting server version

### PR DESCRIPTION
**Pull request by richvdh**
cf #4878

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -44,22 +44,25 @@ those (please be careful to remove any personal or private data). Please surroun
...
 <!--
 What version of Synapse is running?
-You can find the Synapse version by inspecting the server headers (replace matrix.org with
-your own homeserver domain):
-$ curl -v https://matrix.org/_matrix/client/versions 2>&1 | grep "Server:"
+You can find the Synapse version with this command:
+
+$ curl http://localhost:8008/_synapse/admin/v1/server_version
+
+(You may need to replace `localhost:8008` if synapse is not configured to
```
Comment (anoadragon453) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]:
```suggestion
(You may need to replace `localhost:8008` if Synapse is not configured to
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 2]
```diff
@@ -44,22 +44,25 @@ those (please be careful to remove any personal or private data). Please surroun
...
+listen on that port.)
```
Comment (anoadragon453) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]:
```suggestion
listen on that port).
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 3]
```diff
@@ -44,22 +44,25 @@ those (please be careful to remove any personal or private data). Please surroun
 <!-- IMPORTANT: please answer the following questions, to help us narrow down the problem -->
 
 <!-- Was this issue identified on matrix.org or another homeserver? -->
-- **Homeserver**: 
+- **Homeserver**:
 
 If not matrix.org:
 
 <!--
 What version of Synapse is running?
```
Comment (anoadragon453) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]:
```suggestion
 What version of Synapse is running?

```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md (hunk 2 above)
Comment (richvdh) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md]: If the whole sentence is inside the parens, shouldn't the full-stop be? :man_shrugging:

**Pull request merged**